UI/window-basic-auto-config.cpp

```diff
@@ -552,6 +556,37 @@ void AutoConfigStreamPage::ServiceChanged()
 	UpdateCompleted();
 }
 
+void AutoConfigStreamPage::UpdateMoreInfoLink()
+{
+	const char *more_info_link;
+	QString serviceName;
+
+	if (IsCustomService()) {
+		ui->moreInfoButton->hide();
+		return;
+	}
+
+	serviceName = ui->service->currentText();
+	obs_properties_t *props = obs_get_service_properties("rtmp_common");
+	obs_property_t *services = obs_properties_get(props, "service");
+
+	OBSData settings = obs_data_create();
+	obs_data_release(settings);
+
+	obs_data_set_string(settings, "service", QT_TO_UTF8(serviceName));
+	obs_property_modified(services, settings);
+
+	more_info_link = obs_data_get_string(settings, "more_info_link");
+
+	if ((!more_info_link) || (*more_info_link == '\0')) {
+		ui->moreInfoButton->hide();
+	} else {
+		ui->moreInfoButton->setTargetUrl(QUrl(more_info_link));
+		ui->moreInfoButton->show();
+	}
+	obs_properties_destroy(props);
+}
+
 void AutoConfigStreamPage::UpdateKeyLink()
 {
 	if (IsCustomService()) {
```

Comment:
Change (serviceName == "YouTube - RTMP") || (serviceName == "YouTube - HLS") to serviceName.startsWith("YouTube").